src/reducers/vms.js

```diff
@@ -59,10 +52,8 @@ function vms (state, action) {
   logDebug(`The 'vms' reducer action=${JSON.stringify(hidePassword({ action }))}`)
 
   switch (action.type) {
-    case 'UPDATE_VM':
+    case 'UPDATE_VMS':
```

Comment:
where is the respective actionCreator change corresponding to this `UPDATE_VM -> UPDATE_VMS` change?